src/SmartComponents/Reports/Reports.js

```diff
@@ -75,35 +73,41 @@ export const Reports = () => {
     profiles = profilesFromEdges(data);
     error = undefined;
     loading = undefined;
-    showView = profiles && profiles.length > 0;
   }
 
-  return (
-    <>
-      <ReportsHeader />
-      <StateViewWithError stateValues={{ error, data, loading }}>
-        <StateViewPart stateKey="loading">
-          <section className="pf-v5-c-page__main-section">
-            <SkeletonTable colSize={3} rowSize={10} />
-          </section>
-        </StateViewPart>
-        <StateViewPart stateKey="data">
-          <section className="pf-v5-c-page__main-section">
-            {showView ? (
-              <ReportsTable {...{ profiles }} />
-            ) : (
-              <ReportsEmptyState />
-            )}
-          </section>
-        </StateViewPart>
-      </StateViewWithError>
-    </>
-  );
+  return <ReportsBase {...{ data: profiles, error, loading, refetch }} />;
+};
+
+const ReportsWithRest = () => {
+  const params = {
+    params: ['', 100],
+  };
+  const location = useLocation();
+
+  let { data: { data } = {}, error, loading, refetch } = useReports(params);
```

Comment:
```suggestion
  const location = useLocation();

  let { data: { data } = {}, error, loading, refetch } = useReports();
```

since from what I see in the implementation of useReports: it doesn't support any params.